src/Main.hs

```diff
@@ -25,6 +27,11 @@ import Database.SQLite.Simple.ToField (ToField (toField))
 import Foreign.C.Types (CTime (CTime))
 import GHC.Generics (Generic)
 import Network.HTTP.Types (status201, status401, status409)
+import Data.Text hiding (map)
+import Database.SQLite.Simple
+import Database.SQLite.Simple.ToField (ToField (toField))
+import GHC.Generics (Generic)
+import Network.HTTP.Types (status401, status404)
```

Comment:
Bitte duplicates entfernen und status404 oben hinzufügen

Reply:
Should get resolved by 9d8ac164eaaf6e8355e1254ae071ac5eb29b6aef